InheritDoc/InheritDoc/Program.cs

```diff
@@ -1,76 +0,0 @@
-﻿/*
```

Comment:
These files had to be deleted as they were conflicting with the new `csproj` style, which basically just includes all files in its directory. I guess they were probably more just for testing purposes or something..?

I've added this disclaimer into the `InheritDoc/Program.cs`.